modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -208,7 +208,9 @@ class Create_Timepoint extends \NDB_Form
                         WHERE CenterID =:cid",
                         array('cid' => $siteID)
                     );
-                    $psc_labelOptions[$siteID] = $center['Name'];
+                    if (!is_null($center)) {
```

Comment:
Another theme in this PR is verifying that DB results are not `null`. Since [the `pselect*` family does not have any return types](https://github.com/aces/Loris/issues/4284) it is possible for them to return `null` instead of the expected `array` of data. This causes phan to get upset when we try to access an array key without first ensuring that the DB result is not null.